ur_robot_driver/include/ur_robot_driver/ros/hardware_interface.h

```diff
@@ -242,7 +242,7 @@ class HardwareInterface : public hardware_interface::RobotHW
   std::vector<std::string> joint_names_;
   int32_t robot_mode_;
   int32_t safety_mode_;
-
+  
```

Comment:
Please do not introduce whitespace lines. This is probably the reason the format-check fails.

Reply:
I removed it, and I will keep that in mind.